### PR TITLE
Update Web Basic template for beta7. Add missing Logging.Debug depend…

### LINKED ISSUE #316: ASP.NET WebApp (No Auth) for beta7
Feed used: https://www.myget.org/F/aspnetliterelease/api/v2 
 Samples are here: https://onedrive.live.com/redir?resid=43459A6389A804D9%21102054

### PR DESCRIPTION
**Pull request by Spinarooni**
…ency. Closes #316.

### CODE REVIEW

File: templates/projects/webbasic/project.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
+    "Microsoft.AspNet.Mvc.TagHelpers": "6.0.0-beta7",
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta7",
+    "Microsoft.AspNet.Server.Kestrel": "1.0.0-beta7",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta7",
+    "Microsoft.AspNet.StaticFiles": "1.0.0-beta7",
+    "Microsoft.AspNet.Tooling.Razor": "1.0.0-beta7",
+    "Microsoft.Framework.Configuration.Json": "1.0.0-beta7",
+    "Microsoft.Framework.Logging": "1.0.0-beta7",
+    "Microsoft.Framework.Logging.Console": "1.0.0-beta7",
+    "Microsoft.Framework.Logging.Debug" : "1.0.0-beta7"
```
Comment (peterblazejewicz) [inline templates/projects/webbasic/project.json]: @Spinarooni 
Was this package the only change you've made compared to MS sample project? 
Thanks!

**Comment by Spinarooni** [comment]
@peterblazejewicz Yes, I only added:

`"Microsoft.Framework.Logging.Debug" : "1.0.0-beta7"`

It would not build without.

**Comment by peterblazejewicz** [comment]
@Spinarooni 
OK. I think you've been mislead by misformatted code error in downloaded template:
https://github.com/aspnet/Tooling/issues/162
Please rewrite/update your PR to match packages order to avoid another future update:
From:
```
"Microsoft.Framework.Logging": "1.0.0-beta7",
"Microsoft.Framework.Logging.Console": "1.0.0-beta7",
"Microsoft.Framework.Logging.Debug" : "1.0.0-beta7"
```
to:
```
"Microsoft.Framework.Logging": "1.0.0-beta7",
"Microsoft.Framework.Logging.Debug" : "1.0.0-beta7",
"Microsoft.Framework.Logging.Console": "1.0.0-beta7"
```
:)
Thanks!

**Comment by Spinarooni** [comment]
@peterblazejewicz Updated PR.

**Comment by peterblazejewicz** [comment]
@Spinarooni Thanks!
/cc @sayedihashimi

**Comment by sayedihashimi** [comment]
Thanks merged it

**Pull request merged**